debian/snapcraft.manpages

```diff
@@ -0,0 +1 @@
+manpages/snapcraft.1
```

Comment:
This is of course specific to the Debian package build.  If you can, I think it would be better to install the file in setup.py (perhaps in `data_files`), so that it can apply more easily to other distributions; if you do this right it shouldn't be necessary to touch `debian/` at all.